cmd/node/main.go

```diff
@@ -15,14 +17,45 @@ var (
 	commit    string
 	branch    string
 	noMainNet string
+	rootCmd   = &cobra.Command{
+		Use:   "go-spacemesh",
+		Short: "Start spacemesh service",
+	}
 )
 
 func main() { // run the app
 	cmd.Version = version
 	cmd.Commit = commit
 	cmd.Branch = branch
 	cmd.NoMainNet = noMainNet == "true"
-	if err := node.GetCommand().Execute(); err != nil {
+
+	rootCmd.AddCommand(node.GetNodeServiceCommand())
```

Comment:
As we aim to have 3 commands:
- `./go-spacemesh` (no command) runs monolith smeshing + node services (everything)
- `./go-spacemesh smesing-service`: runs smeashing-service (no p2p etc)
- `./go-spacemesh node-service`: runs node-service (no atx building etc.)

I think it would be easier to leave the `node.go` untouched instead of moving it to node_service.go. Later the necessary contents can be moved to `node_service.go`, wdyt?

Reply:
Done.